remme/shared/forms/_validators.py

```diff
@@ -0,0 +1,23 @@
+from wtforms import validators
+
+
+class TypeRequired(object):
+    """
+    Validates the address for the required type.
```

Comment:
If it validate address, so `AddressTypeRequired`.

Reply:
@dmytrostriletskyi, done.